Cargo.toml

```diff
@@ -29,6 +29,7 @@ log4rs = "1.3.0"
 flate2 = {version = "1.0.35", optional = true}
 tar = {version = "0.4.43", optional = true}
 minreq = { version = "2.12.0", features = ["https"] , optional = true}
+rand = "0.9.0"
```

Comment:
rust bitcoin already comes with a rand crate. can we use that instead of a new dep?

Reply:
I just removed that crate and used the one you suggested.